### PR TITLE
🚮 Clean up ANALYTICS_VENDOR_SPLIT

### LINKED ISSUE #22757: [analytics] cleanup vendor config JS files and gulp task
## Describe the new feature or change to an existing feature you'd like to see

Cleanup the vendor config JS files, vendors.js and related files as well as the gulp task `generate-vendor-configs`.

## Additional context

We are migrating all vendor config JS files to JSON files. We will have a gulp task to do this while we are running AB tests so the JS and JSON files do not go out of sync. After these AB tests are complete and we finish migration, we will need to cleanup these unnecessary files.

cc @zhouyx

### PR DESCRIPTION
**Pull request by zhouyx**
Closes #22757

Any advice on preserving the git history?

### CODE REVIEW

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 1]
```diff
@@ -135,7 +135,7 @@
...
     "social": "https://www.google-analytics.com/collect?v=1&_v=a1&ds=AMP&aip&_s=1&dt=_title_&sr=_screen_width_x_screen_height_&_utmht=_timestamp_&cid=_client_id_&tid=!account&dl=_source_url_&dr=_external_referrer_&sd=_screen_color_depth_&ul=_browser_language_&de=_document_charset_&t=social&jid=&sa=!socialAction&sn=!socialNetwork&st=!socialTarget&a=_page_view_id_&z=_random_",
     "timing": "https://www.google-analytics.com/collect?v=1&_v=a1&ds=AMP&aip&_s=1&dt=_title_&sr=_screen_width_x_screen_height_&_utmht=_timestamp_&cid=_client_id_&tid=!account&dl=_source_url_&dr=_external_referrer_&sd=_screen_color_depth_&ul=_browser_language_&de=_document_charset_&t=!timingRequestType&jid=&plt=_page_load_time_&dns=_domain_lookup_time_&tcp=_tcp_connect_time_&rrt=_redirect_time_&srt=_server_response_time_&pdt=_page_download_time_&clt=_content_load_time_&dit=_dom_interactive_time_&a=_page_view_id_&z=_random_&aae=true",
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: I think `aae=true` has to be removed here too: https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/config.js#L150:L154

Comment (zhouyx) [reply extensions/amp-analytics/0.1/test/vendor-requests.json]: The `addExperimentParams_()` function is removed : )

Comment (micajuine-ho) [reply extensions/amp-analytics/0.1/test/vendor-requests.json]: @zhouyx Sorry, what I meant to say was: `aae=true` should be removed from request `timing` as well

**Comment by zhouyx** [comment]
@erwinmombay @estherkim 

Do you know how to fix the lint error? Thanks
```
extensions/amp-analytics/0.1/vendors.js
  99:3   error  Cannot export side-effect                                                                                                                                    local/no-export-side-effect
```

File: extensions/amp-analytics/0.1/test/vendor-requests.json (hunk 1 above)
Comment (zhouyx) [reply extensions/amp-analytics/0.1/test/vendor-requests.json]: Ah! you're right! Nice catch!

**Pull request merged**